src/services/edusharing/logic/connector.js

```diff
@@ -0,0 +1,231 @@
+const REQUEST_TIMEOUT = 8000; // ms
+const request = require('request-promise-native');
+const { Configuration } = require('@schul-cloud/commons')
+
+const DOMAIN = Configuration.get('EDUSHARING__DOMAIN');
+const USER = Configuration.get('EDUSHARING__USER');
+const PASSWORD = Configuration.get('EDUSHARING__PASSWORD');
+const GRANT_TYPE = Configuration.get('EDUSHARING__GRANT_TYPE');
+const OAUTH_SECRET = Configuration.get('EDUSHARING__OAUTH_SECRET');
+const CLIENT_ID = Configuration.get('EDUSHARING__CLIENT_ID');
+
+
+// STACKOVERFLOW BEAUTY
+function validURL(str) {
+	const pattern = new RegExp(
+		'^(https?:\\/\\/)?'
+		+ '((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.)+[a-z]{2,}|'
+		+ '((\\d{1,3}\\.){3}\\d{1,3}))'
+		+ '(\\:\\d+)?(\\/[-a-z\\d%_.~+]*)*'
+		+ '(\\?[;&a-z\\d%_.~+=-]*)?'
+		+ '(\\#[-a-z\\d_]*)?$',
+		'i'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing trailing comma.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4991925)

Reply:
at least for the bbb in videoconference/logic/utils this already exists :)